app/com/gu/floodgate/reindex/ProgressTracker.scala

```diff
@@ -133,20 +134,14 @@ class ProgressTracker(ws: WSClient, runningJobService: RunningJobService, jobHis
           runningJob.rangeFrom,
           runningJob.rangeTo
         )
-        val periodIndexing = Minutes.minutesBetween(runningJobUpdate.startTime, DateTime.now())
 
-        val jobHasStalled: Boolean = runningJobUpdate.documentsIndexed == 0 && (periodIndexing.getMinutes - 10) >= 0
```

Comment:
Are we removing this because it would marked as failed content jobs for which there isn't yet an estimate for the number of expected documents? 

With this change, is it now up to the user to assess if a job has failed and cancel it accordingly? (I prefer this BTW).

Reply:
That's the aim, yes. This application shouldn't be deciding what is happening inside the service generating the reindex messages I think.